src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -217,6 +216,13 @@ parseBlock (Elem e) = do
              else divWith (attrValue "id" e, ["caption"], []) <$> wrapWithHeader 6 (getBlocks e)
         "fn-group" -> parseFootnoteGroup
         "ref-list" -> parseRefList e
+        "alternatives" -> if hasFormulaChild e
```

Comment:
Given these can now be considered as block elements, as per the refinement of `isBlockElement`, the respective parse cases needed adding.